rust/connlib/tunnel/src/client.rs

```diff
@@ -1621,6 +1621,29 @@ fn maybe_mangle_dns_response_from_cidr_resource(
     packet
 }
 
+fn truncate_dns_response(message: &Message<Vec<u8>>) -> Vec<u8> {
+    let mut message_bytes = message.as_octets().to_vec();
+
+    if message_bytes.len() > MAX_DATAGRAM_PAYLOAD {
+        tracing::debug!(?message, message_length = %message_bytes.len(), "Too big DNS response, truncating");
```

Comment:
Need to test for this in the workflow file.

Reply:
oops, forgot to go up 1 directory to commit my bad